.github/workflows/build.yml

```diff
@@ -64,20 +61,23 @@ jobs:
         run: dotnet restore --locked-mode
         shell: pwsh
 
+      - name: Remove SQL proj
+        run: dotnet sln bitwarden-server.sln remove src/Sql/Sql.sqlproj
+
       - name: Build OSS solution
-        run: msbuild bitwarden-server.sln /p:Configuration=Debug /p:DefineConstants="OSS" /verbosity:minimal
+        run: dotnet build bitwarden-server.sln -p:Configuration=Debug -p:DefineConstants="OSS" --verbosity minimal
         shell: pwsh
 
       - name: Build solution
-        run: msbuild bitwarden-server.sln /p:Configuration=Debug /verbosity:minimal
+        run: dotnet build bitwarden-server.sln -p:Configuration=Debug --verbosity minimal
         shell: pwsh
 
       - name: Test OSS solution
-        run: dotnet test ./test --configuration Debug --no-build --logger "trx;LogFileName=oss-test-results.trx" || true
+        run: dotnet test ./test --configuration Debug --no-build --logger "trx;LogFileName=oss-test-results.trx"
         shell: pwsh
 
       - name: Test Bitwarden solution
-        run: dotnet test ./bitwarden_license/test --configuration Debug --no-build --logger "trx;LogFileName=bw-test-results.trx" || true
+        run: dotnet test ./bitwarden_license/test --configuration Debug --no-build --logger "trx;LogFileName=bw-test-results.trx"
```

Comment:
@withinfocus Any reason we did `|| true` here initially?

Reply:
You want to let the test run failure still continue the process here to catch said test failures. The reporter will also emit a failure to the Action.

Example: https://www.octopus.com/blog/githubactions-running-unit-tests#:~:text=The%20test%20command%20returns%20a%20non%2Dzero%20exit%20code%20if%20any%20tests%20fail%2C%20but%20you%20defer%20responsibility%20for%20responding%20to%20failed%20tests%20to%20the%20test%20processor.

Reply:
That's what the `if always()` is for, per https://github.com/dorny/test-reporter#recommended-setup-for-public-repositories right?

Reply:
What I believe I saw is this stoped the process, but I see you trying it out with a failed test -- open to the change.

Reply:
@withinfocus https://github.com/bitwarden/server/runs/10932209501 seems to work.

Reply:
Good catch!